src/Modifier/StreamIterator.php

```diff
@@ -156,18 +156,17 @@ public function setFlags($flags)
      * @param array  $fields
      * @param string $delimiter
      * @param string $enclosure
-     * @param string $escape
      *
      * @return int
      */
-    public function fputcsv(array $fields, $delimiter = null, $enclosure = null, $escape = null)
+    public function fputcsv(array $fields, $delimiter = ',', $enclosure = '"', $escape = '\\')
```

Comment:
it should be part of bugfix because it aligns the StreamIterator to how SplFileObject and fputcsv works. StreamIterator must be align to SplFileObject and it did no align.